src/slicer.h

```diff
@@ -136,7 +136,11 @@ class SlicerLayer
     /*!
      * Connecting polylines that are not closed yet.
      *
-     * Any polylines that are closed by this function are added to this->polygons.
+     * Any polylines that are closed by this function are added to
+     * this->polygons.  All possible polyline joins that meet the
+     * distance and reversal criteria will be performed.  This
+     * function will not introduce any copies of the same polyline
+     * segment.
```

Comment:
Just to be clear: I was asking for this because there is an option to also print the open_polylines.

`open_polylines` is not just a local temp variable to compute `this->polygons`